byzcoin/README.md

```diff
@@ -14,6 +14,43 @@ them as seem fit.
 This document describes the part of ByzCoin that are implemented and how to
 use them. It should grow over time as more parts of the system are implemented.
 
+## Short summary of ByzCoin
+
+- ByzCoin is a **distributed ledger**: a database that is
+  distributed across several nodes, and where the authority to add changes
+  to the database is decentralized.
+- To ensure that updates to the database are strictly ordered, ByzCoin
+  uses a **blockchain** called a **Skipchain**: every node has its local copy of the ledger
+  (database). Every change to the ledger is packed into a block that is
+  cryptographically chained to the previous one, and forward links in
+  the chain represent consensus among the verifiers that these changes
+  belong in the database.
+- In ByzCoin, the data are organised as instances of **Smart
```

Comment:
```suggestion
- In ByzCoin, the data is organised as instances of **Smart
```

Reply:
that's a debate, but it seems there is nothing wrong with this plural form of "data".

Reply:
I was not aware that there is a discussion around this - https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular

According to that article you're more of a traditionalist, though ;)